src/ui/public/notify/notifier.js

```diff
@@ -310,48 +320,69 @@ Notifier.prototype._showFatal = function (err) {
  * @param  {Error|String} err
  * @param  {Function} cb
  */
-Notifier.prototype.error = function (err, cb) {
-  return add({
+Notifier.prototype.error = function (err, opts, cb) {
+  const overrideable = ['lifetime', 'icon'];
```

Comment:
BTW, both spellings are acceptable 😄 but "overridable" is more [in vogue](https://books.google.com/ngrams/graph?content=overridable%2C+overrideable&year_start=1800&year_end=2000&corpus=15&smoothing=3&direct_url=t1%3B%2Coverridable%3B%2Cc0%3B.t1%3B%2Coverrideable%3B%2Cc0).